packages/smart-wallet/src/utils.js

```diff
@@ -1,10 +1,12 @@
 /* eslint-disable no-undef-init */
-import { deeplyFulfilledObject, objectMap } from '@agoric/internal';
+import { deeplyFulfilledObject, objectMap, makeTracer } from '@agoric/internal';
 import { observeIteration, subscribeEach } from '@agoric/notifier';
 import { E } from '@endo/far';
 
 export const NO_SMART_WALLET_ERROR = 'no smart wallet';
 
+const trace = makeTracer('WUTIL', false);
```

Comment:
the `warn` and `error` sent to STDERR but this will send to STDOUT. I suppose now that it's disabled by default whoever enables it will understand the this can mix with any JSON output. (I don't recall whether this command has any)